docs/reference/content/driver-async/tutorials/ssl.md

```diff
@@ -155,3 +155,26 @@ command line program.
 For more information on configuring a Java application for TLS/SSL, please
 refer to the [`JSSE Reference Guide`](http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSS
 ERefGuide.html).
+
+## JVM Support for TLS v1.1 and newer
+
+Industry best practices recommend, and some regulations require, the use of TLS 1.1 or newer. Though no application changes are required
+for the driver to make use of the newest TLS protocols, Java runtime environments prior to Java 8 started to enable TLS 1.1 only in later
+updates:
+
+* Java 7
+  - TSL 1.1 was first enabled by default in
+    [Update 131](http://www.oracle.com/technetwork/java/javaseproducts/documentation/javase7supportreleasenotes-1601161.html#R170_131),
+    released October 8, 2016.
+  - TLS 1.1 was first enabled via `jdk.tls.client.protocols` system property in
```

Comment:
This reads to me like no changes are required to make use of this, just upgrade to that version of Java 7 or newer. But I think developers have to actually enable TLSv1.1 using this property, right? Maybe say something like "Starting with Java 7 update 95, TLSv1.1 can be enabled using ..."